src/templates/static/platforms.html.tt

```diff
@@ -53,138 +53,138 @@
+                            <p class="center-text">[% l("MetaTrader 5 is an institutional multi-asset platform offering outstanding trading possibilities and technical analysis tools. It allows trading Foex, Stocks, Futures, CFDs and it provides everything needed for full-fledged and flexible trading operations.") %]</p>
+                            <div class="gr-row gr-padding-30">
+                                <div class="gr-6 gr-12-m gr-padding-30 gr-parent">
+                                    <img class="responsive" src="[% request.url_for('images/pages/home/mt5.png') %]" />
+                                </div>
+                                <div class="gr-5 gr-push-1 gr-12-m gr-push-0-m">
+                                    <h3>[% l("MetaTrader 5") %]</h3>
+                                    <p>[% l("[_1] enters Forex and CFD trading with the universally acclaimed MetaTrader 5 to its continued leadership presence in the binary options market.", website_name) %]</p>
+                                    <p>[% l("Trade Forex and Contracts for Difference (CFD) assets with highly-competitive leverage and zero commission.") %]</p>
+                                    <p>
+                                        <a class="button" href="[% request.url_for('user/metatrader') %]">
+                                            <span>[% l("Create MetaTrader 5 Account in [_1]", website_name) %]</span>
+                                        </a>
+                                    </p>
+                                </div>
                             </div>
                         </div>
-                    </div>
 
-                    <div class="container gr-padding-30">
-                        <h2 class="center-text gr-padding-20">[% l("Trade with a powerful interface provided") %]</h2>
-                        <div class="gr-row gr-padding-30 gr-parent">
-                            <div class="gr-6 gr-12-m">
-                                <img class="responsive" src="[% request.url_for('images/pages/platforms/mt5/interface.svg') %]" alt="" />
+                        <div class="gr-padding-10 fill-bg-color">
+                            <div class="container gr-padding-30">
+                                <h2 class="center-text">[% l("Enhanced Trading with MetaTrader 5") %]</h2>
+                                <div class="gr-row center-text enhanced">
+                                    [% INCLUDE enhanced_items image='trade'      text=l("Trade Forex and CFD asset classes with an award winning broker") %]
+                                    [% INCLUDE enhanced_items image='leverage'   text=l("Trade the Forex markets with leverage up to 100:1, and Volatility Indices with leverage up to 500:1") %]
+                                    [% INCLUDE enhanced_items image='demo'       text=l("Open a free Demo account. Upgrade to a Real account by depositing funds thru your Binary.com account cashier") %]
```

Comment:
minor typo here: `thru` should be through :-)

Reply:
Fixed, thanks.